itest/lnd_open_channel_test.go

```diff
@@ -621,3 +621,38 @@ func verifyCloseUpdate(chanUpdate *lnrpc.ChannelEventUpdate,
 
 	return nil
 }
+
+// testFundingExpiryBlocksOnPending checks that after an OpenChannel, and
+// before the funding transaction is confirmed, that the FundingExpiryBlocks
+// field of a PendingChannels decreases.
+func testFundingExpiryBlocksOnPending(ht *lntest.HarnessTest) {
+	alice, bob := ht.Alice, ht.Bob
+	param := lntest.OpenChannelParams{Amt: 100000}
+	update := ht.OpenChannelAssertPending(alice, bob, param)
+
+	// At this point, the channel's funding transaction will have been
+	// broadcast, but not confirmed. Alice and Bob's nodes should reflect
+	// this when queried via RPC. FundingExpiryBlock should decrease
+	// as blocks are mined, until the channel is confirmed. Empty blocks
+	// won't confirm the funding transaction, so let's mine a few empty
+	// blocks and verify the value of FundingExpiryBlock at each step.
+	const numEmptyBlocks = 3
+	for i := int32(0); i < numEmptyBlocks; i++ {
+		expectedVal := funding.MaxWaitNumBlocksFundingConf - i
+		pending := ht.AssertNumPendingOpenChannels(alice, 1)
+		require.Equal(ht, expectedVal, pending[0].FundingExpiryBlocks)
+		pending = ht.AssertNumPendingOpenChannels(bob, 1)
+		require.Equal(ht, expectedVal, pending[0].FundingExpiryBlocks)
+		ht.MineEmptyBlocks(1)
+	}
+
+	// Mine 1 block to confirm the funding transaction. At this point, the
+	// channel should be fully opened and there should be no pending
+	// channels remaining for either node.
+	ht.MineBlocks(1)
+	ht.AssertNumPendingOpenChannels(alice, 0)
+	ht.AssertNumPendingOpenChannels(bob, 0)
```

Comment:
@yyforyongyu , the build failure seems strange. This test passes when run locally and in isolation. I'm testing if the complete `make itest` passes locally. Maybe it's a test isolation issue. That being said though, is there some difference between the local run and the build run?  The failure seems to be after mining a single block and checking for the number of pending open channels (the lines above). On a local run, mining one block moves the channel from pending to open. In the build environment, is this still the case?

In any event, the intention of the test is to check the decreasing value of `FundingExpiryBlocks` and not that channels transition from pending to open after mining a block. I'll check if the test still passes locally without these lines, and I think it should be fine to remove them altogether. Thoughts?